kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/UserCredentialsForm.vue

```diff
@@ -286,12 +301,10 @@
         context:
           'The learner is creating their account for an existing facility and is told what that is',
       },
-      /* eslint-disable kolibri/vue-no-unused-translations */
-      signInInstead: {
+      SignInInstead: {
```

Comment:
Will need to leave this as camelCase rather than PascalCase, as otherwise it will count this as a new string.